tests/copilot_helper/conftest.py

```diff
@@ -36,6 +36,9 @@ def fakefs(fs):
     fs.add_real_file(BASE_DIR / "dbt_copilot_helper/addons-template-map.yml")
     fs.add_real_directory(Path(jsonschema.__path__[0]) / "schemas/vocabularies")
 
+    # To avoid 'Could not find a suitable TLS CA certificate bundle...' error
+    fs.add_real_file(BASE_DIR / "venv/lib/python3.11/site-packages/certifi/cacert.pem")
+
     return fs
```

Comment:
I wonder if there's a better way of finding the cacert. This will break when we upgrade python. Perhaps look in `venv/lib` for the most recent python and use that. Would prob need to be done outside of this function though as I think only the fakefs is visible inside the function.